services/slackServices/shoutout.helpers.js

```diff
@@ -10,6 +10,7 @@ exports.saveToDatabase = async dbInfo => {
     giver_id: giver.id,
     receiver_id: reciever.id,
     message: dbInfo.message,
+    timestamp: dbInfo.timestamp,
```

Comment:
yo, since we have the other pr handling adding timestamps, how about we remove the timestamp property what do you think